src/renderer/entities/network/model/network-model.ts

```diff
@@ -39,6 +40,53 @@ const $apis = createStore<Record<ChainId, ApiPromise>>({});
 const $connections = createStore<Record<ChainId, Connection>>({});
 const $connectionStatuses = createStore<Record<ChainId, ConnectionStatus>>({});
 
+// not sure where this should live
+// network-utils did not seem like the best place
+const getExtendedChain = (
+  chains: Chain[],
+  connections: Record<`0x${string}`, Connection>,
+  connectionStatuses: Record<`0x${string}`, ConnectionStatus>,
+) => {
+  return chains.reduce<ExtendedChain[]>((acc, chain) => {
+    const connection = connections[chain.chainId];
+    const extendedChain = {
+      ...chain,
+      connection,
+      connectionStatus: connectionStatuses[chain.chainId],
+    };
+
+    acc.push(extendedChain);
+
+    return acc;
+  }, []);
+};
+
+const $activeChainsSorted: Store<ExtendedChain[]> = combine(
```

Comment:
These stores should be located in a dedicated feature - `src/features/network/NetworksList`

Reply:
Makes sense, thanks.